### DISCUSSION TITLE
Qualcomm Atheros QCA9377 - Test results

### DISCUSSION THREAD

**Discussion by Paulinhoice** [vanilla-wiiu/vanilla#20] [Hardware Results]
The Qualcomm Atheros QCA9377 paired and connected very easily. I was able to get the controls and audio working, but the video remained on a black screen.

The strange thing was that after using it for a short time (less than 1 minute, probably around 30 to 40 seconds), it lost the connection to the Wii U (the Wi-Fi is managed by itself).

If you want to see my terminal, follow the link below:

https://pastebin.com/zvAvSG5K

**Comment by VoxelTek** [comment]
Test results have been added to the [Wireless Compatibility](https://github.com/vanilla-wiiu/vanilla/wiki/Wireless-Compatibility) page 👍🏻

**Comment by GabubuAvailable** [comment]
I have the same WiFi chip, tested on Linux Mint and I managed to sync it but no image and no sound coming from it and cannot control it. Tested on Lenovo Ideapad 100-15IBR. So I can also confirm that it partially work.
